Packs/SecurityTrails/Integrations/SecurityTrails/SecurityTrails.py

```diff
@@ -0,0 +1,1069 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' CONSTANTS '''
+
+removed_keys = ['endpoint', 'domain', 'hostname']
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+
+    def handle_error(self, message: dict = {}):
+        return_error(message['message'])
+
+    def domain_tags(self, hostname: str = None):
+        res = self._http_request(
+            'GET',
+            f'domain/{hostname}/tags',
+            ok_codes=[200, 403],
+            resp_type="response"
```

Comment:
if you eventually use res.json() for each response, why change the resp_type?
the `_http_request` already takes care of it.

Reply:
Because the _http_request does not parse the error message in the response. I had to force the response type to be a response because the handling of the error message in the function on L11